core/src/integration-test/java/io/aiven/kafka/tieredstorage/RemoteStorageManagerTest.java

```diff
@@ -401,15 +404,20 @@ void testRequiresCompression(final CompressionType compressionType, final boolea
 
         // Configure the RSM.
         final int chunkSize = 1024 * 1024;
-        final Map<String, ?> config = new HashMap<>(Map.of(
-            "chunk.size", Integer.toString(chunkSize),
-            "storage.backend.class",
-            "io.aiven.kafka.tieredstorage.storage.filesystem.FileSystemStorage",
-            "key.prefix", "test/",
-            "storage.root", targetDir.toString(),
-            "compression.enabled", "true",
-            "compression.heuristic.enabled", "true"
-        ));
+        final Map<String, ?> config = new HashMap<>() {{
+                put("chunk.size", Integer.toString(chunkSize));
+                put("storage.backend.class",
+                    "io.aiven.kafka.tieredstorage.storage.filesystem.FileSystemStorage");
+                put("key.prefix", "test/");
+                put("storage.root", targetDir.toString());
+                put("compression.enabled", "true");
+                put("compression.heuristic.enabled", "true");
+                put("chunk.cache.size", 10000);
+                put("chunk.cache.path", tmpDir.toString());
```

Comment:
[Nit] This is unnecessary since we're using the in-mem cache

Reply:
Fixed.